test/test_auth.py

```diff
@@ -54,22 +54,42 @@ def test_auth_dbname_disabled(bouncer):
         bouncer.test(dbname="pauthz", user="someuser", password="anypasswd")
 
 
-def test_auth_dbname_with_auto_database(bouncer):
```

Comment:
I think this previously existing test was testing something slightly differently. The auto-db with an auth_dbname on the same line. So I think it makes sense to keep it.